core/testdata/testspecs/v1_specs.go

```diff
@@ -0,0 +1,40 @@
+package testspecs
+
+var (
+	RandomnessJob = `
```

Comment:
@spooktheducks @samsondav @j16r I think we should move everything into this format instead of using actual files for a few reasons:
1. Path independence. Tests using this need not specify the path to the package
2. Possible generation/templating. If its all in go we can add templated versions, functions etc. to create all types of specs

Reply:
Makes sense to me